core/domain/protofiles/training_job_data.proto

```diff
@@ -0,0 +1,33 @@
+// coding: utf-8
+//
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "core/domain/protofiles/text_classifier.proto";
+
+message TrainingJobData {
+  message JobResult {
+    string job_id = 1;
+    // Each of the classifier algorithms' proto message must be present in
+    // the oneof classifier_data field.
+    oneof classifier_data {
```

Comment:
Prefer to avoid the word ``data`` in general, if you can help it. Could we call this ``classifier_model``, instead?

Reply:
Also ditto above--can we get more specific than ``TrainingJobData``?

Reply:
I will change `classifier_data` to `classifier_frozen_model`. For the `TrainingJobData`, I am thinking of changing it to `TrainingJobResponsePayload`?

Reply:
Done.

Reply:
Both sound good--thanks!